### DISCUSSION TITLE
Changing Tabs Settings

### DISCUSSION THREAD

**Discussion by Buko0** [zen-browser/desktop#4065] [Ideas]
Imagine that Zen Browser offers a customizable setting that allows you to open new tabs at the top instead of the bottom. This would be an adjustable option, so users who prefer tabs at the bottom can keep that setting, while others can choose to open their tabs at the top. I believe that if such a feature were introduced, most people would likely opt for the top setting, but that's just a guess

Edit: They added a feature that allows user to place the new tab button to the top of the tabs menu by customizing toolbar.
My special thanks <3

**Comment by dderg** [comment]
Especially given that you can move "new tab" button to the top, you would expect it would create new tabs at the top too

**Comment by mr-cheffy** [comment]
In the new update, if the "newtab button at the top" is on, it will start adding tabs belllow the pinned tabs

**Reply by Buko0** [reply]
U guys are the best
